.cirrus.yml

```diff
@@ -82,6 +82,22 @@ task:
   << : *COMMON_STEPS_TEMPLATE
 
 # Mac
+task:
```

Comment:
We should change this to conform to the YAML standard. Cirrus CI is permissive about wrong YAML syntax, but duplicate mapping keys are not acceptable in pure YAML.

See https://github.com/dlang/phobos/commit/05c1b59919d1f2f55023ecb38ba1959216ae4235#diff-62587956f943bb2503db7bc6dd27d0d888074a1c0ecaab3f570ad611aff0f7bb and https://twitter.com/codewithanthony/status/1430393351212355585 .